src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/SelectDeliveryModelMapperHelper.cs

```diff
@@ -28,7 +28,7 @@ public async Task<SelectDeliveryModelViewModel> Map(long providerId, string cour
 
         public async Task<bool> HasMultipleDeliveryModels(long providerId, string courseCode)
         {
-            var response = await _client.GetProviderCourseDeliveryModels(providerId, courseCode);
+            var response = await _client.GetProviderCourseDeliveryModels(providerId, courseCode, 0);
```

Comment:
@Pauljgraham the passed parameter needs to be the `accountLegalEntityId` for the apprenticeship being edited.

This is because there is a new Delivery Model available to employers on the FJAA register - so the Id of the ALE is used to determine whether or not they are on the register. The underlying api allows a zero value only for purposes of backwards-compatibility.

Reply:
The same I imagine is true on the employer side

Reply:
It is